rocky/rocky/locale/nl/LC_MESSAGES/django.po

```diff
@@ -77,7 +79,7 @@ msgstr "Selecteer een account type om door te gaan."
 #: account/forms/account_setup.py
 #: onboarding/templates/step_3e_trusted_acknowledge_clearance_level.html
 msgid "Trusted clearance level"
-msgstr "Toegewezen vrijwaring niveau"
+msgstr "Zet vrijwaringsniveau"
```

Comment:
Zet is een handeling, toegewezen was een status. weten we zeker dat dit goed gaat overal?